TF_Tooling_Web/.prettierrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "singleQuote": true,
+    "tabWidth": 4,
+    "endOfLine": "crlf",
+    "printWidth": 120,
+    "importOrder": [
```

Comment:
Probably worth reviewing the `importOrder` I don't think all of these are necessary